docs/settings/storagenetwork/manual.md

```diff
@@ -0,0 +1,309 @@
+
+### Example Configuration
+
+In the following configuration steps, we will use the below configurations for example.
+
+Network Config:
+- ClusterNetwork: test
+- NICs
+  - eth2
+  - eth3
+
+VLANs:
+- Namespace: default
+- Name: storagenetwork-1
+- VLAN ID: 100
+
+IP Range:
+- 192.168.0.0/24
+
+## Configuration
+
+### Step 1: Create VLANs
+
+- Create Cluster Networks/Configs in Web UI.
+    - Choose Networks > Cluster Networks/Configs in the left sidebar.
+- Create VLANs via kubectl command below with desired namespace, name, and VLAN ID, and keep the namespace and name for further configurations.
+- Put Name in name field.
+- Put Namespace in namespace field.
+- Put ClusterNetwork name with `-br` suffix in "bridge" field.
+- Put IP Range into `ipam.range` field.
+
+:::caution
+
+Don't remove the VLAN if Longhorn storage-network is still using that VLAN.
+
+:::
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  annotations:
+    storage-network.settings.harvesterhci.io: "true"
+  name: storagenetwork-1
+  namespace: default
+spec:
+  config: '{"cniVersion":"0.3.1","type":"bridge","dns":{},"bridge":"test-br","promiscMode":true,"vlan":100,"ipam":{"type":"whereabouts","range":"192.168.0.0/24"}}'
+EOF
+```
+
+### Step 2: Shutdown All VMs
+
+- Stop all VMs.
+- Check all VMs are off.
+    - `kubectl get -A vmi`
+- You should not see any existing VMI.
+
+### Step 3: Shutdown All Pods with Longhorn Volumes
+
+:::caution
+
+Please stop pods by order.
+
+:::
+
+#### Stop Rancher Monitoring
+
+- Stop Rancher Monitoring.
+
+```bash
+kubectl patch -n fleet-local managedchart rancher-monitoring --type merge -p '{"spec":{"paused": true}}'
+```
+
+- Check Rancher Monitoring paused is true.
+
+```bash
+kubectl get -n fleet-local managedchart rancher-monitoring -o=jsonpath='{.spec.paused}'
+```
+
+#### Stop Grafana
+
+- Stop Grafana.
+
+```bash
+kubectl patch -n cattle-monitoring-system deployments rancher-monitoring-grafana --type merge -p '{"spec":{"replicas": 0}}'
+```
+
+- Check Deployment replicas are 0.
+
+```bash
+kubectl get -n cattle-monitoring-system deployments rancher-monitoring-grafana -o=jsonpath='{.spec.replicas}'
+```
+
+#### Stop Prometheus
+
+- Stop Prometheus.
+
+```bash
+kubectl patch -n cattle-monitoring-system prometheuses rancher-monitoring-prometheus --type merge -p '{"spec":{"replicas": 0}}'
+```
+
+- Check Prometheus replicas is 0.
+
+```bash
+kubectl get -n cattle-monitoring-system prometheuses rancher-monitoring-prometheus -o=jsonpath='{.spec.replicas}'
+```
+
+#### Stop Harvester VM Import Controller
+
+- Check VM Import Controller is existing.
+- If not, skip this step.
+
+```bash
+kubectl get -n harvester-system deployments harvester-harvester-vm-import-controller
+```
+
+- If VM Import Controller is existing, please stop Harvester VM Import Controller.
```

Comment:
```suggestion
- If VM Import Controller does not exist, please stop Harvester VM Import Controller.
```

Reply:
Do we need to change it to "exists"?

Reply:
Yes please. Sorry I misread it.

Reply:
I fixed it because there is another error.
Thanks a lot